### DISCUSSION TITLE
Is possible to retrieve more tweets by id using get_status?

### DISCUSSION THREAD

**Discussion by thekillgfx** [tweepy/tweepy#1393] [Questions] [answered]
**What would I like to do?**
I would like to retrieve more tweets using the `get_status `function passing as parameter an array of ids.

**Why would I like to do so?**
I'm worried about hitting some API limit, usually when you have to do a lot of requests is a good practice to collect them and conenct to the API once instead of one connection per request.

**Example code I'd like to use**
```
auth = tweepy.OAuthHandler(api_key, api_secret)
auth.set_access_token(access_token, access_secret)
twitter = tweepy.API(auth)

ids = [] #array containing hundreds of ids
tweets = twitter.get_status(ids, tweet_mode='extended')
for tweet in tweets:
    #do my things
```

**Is there the possibility of hitting an API limit?**
I'm planning to request infos about hundreds of tweet every few minutes, can that be done?
Is there any knowed API limit to be should aware of?

**Comment by AFirouzi** [comment]
@thekillgfx There are some limits. you can find them in [twitter documents](https://developer.twitter.com/en/docs/basics/rate-limiting). About your first question, I don't know what you need exactly but you can use [Cursor](http://docs.tweepy.org/en/latest/cursor_tutorial.html) or [API.search](http://docs.tweepy.org/en/latest/api.html#API.search) to get all of the tweets based on your desired query. If you have a list of ids in possession, then for loop is the easiest way for your purpose.

**Comment by Harmon758** [comment, marked as answer]
You should look into using the [GET statuses/lookup endpoint](https://developer.twitter.com/en/docs/tweets/post-and-engage/api-reference/get-statuses-lookup) with [`API.statuses_lookup`](https://tweepy.readthedocs.io/en/latest/api.html#API.statuses_lookup).
This would allow you to retrieve up to 100 Tweets per request by ID, with a rate limit of 900 requests / 15 min. with user auth.
